llvm/test/tools/llvm-readtapi/command-line.test

```diff
@@ -0,0 +1,7 @@
+; RUN: llvm-readtapi --help 2>&1 | FileCheck %s 
+; RUN: llvm-readtapi -help 2>&1 | FileCheck %s 
+
+CHECK: OVERVIEW: LLVM TAPI file reader and manipulator
+CHECK: USAGE: llvm-readtapi [options] <inputs>
+CHECK: OPTIONS:
+CHECK:   -help    display this help
```

Comment:
Does it also emit help for the compare option?

Reply:
Yes, I didn't add the others so I wouldn't need to update it as more options get added.